Gemfile.lock

```diff
@@ -418,4 +422,4 @@ DEPENDENCIES
   whenever (= 0.9.4)
 
 BUNDLED WITH
-   1.11.2
+   1.12.5
```

Comment:
It might mean you end up with a permanently dirty tree if bundler decides to overwrite the BUNDLED_WITH and that means the update_git / update_all scripts in the development repo don't do their thing.  I mean, it's a minor annoyance for sure, but still annoying.